Backend/app/rider-platform/rider-app/Main/src/Storage/Queries/Booking.hs

```diff
@@ -187,12 +189,22 @@ findAssignedByRiderId personId = Esq.buildDType $ do
     pure (booking, fromLoc, mbToLoc, mbTripTerms, mbRentalSlab)
   join <$> mapM buildFullBooking fullBookingsT
 
-findAssignedByQuoteId :: Transactionable m => Id Quote -> m (Maybe Booking)
-findAssignedByQuoteId quoteId = Esq.buildDType $ do
+findAssignedByEstimateId :: Transactionable m => Id Estimate -> m (Maybe Booking)
+findAssignedByEstimateId estimateId = Esq.buildDType $ do
   fullBookingsT <- Esq.findOne' $ do
-    (booking :& fromLoc :& mbToLoc :& mbTripTerms :& mbRentalSlab) <- from fullBookingTable
+    (booking :& fromLoc :& mbToLoc :& mbTripTerms :& mbRentalSlab :& _ :& driverOffer) <-
+      from $
+        fullBookingTable
```

Comment:
> why to join extra data? we're trying to remove all joins. It can impact DB perf.

I think i already sent you my one million join query, which was as fast as any other query? Or you are talking about some other perf param? This query doesn't contain redundant joins, which could slow down things, we need all of them, so it must be faster that separate selects.
But anyway, selects refactoring is not what this PR is about. Currently this join is here, so i'm using it, it should be removed in separate PR (I think it's @roman-bodavskiy  PR?)

Reply:
Sure. Actually, we are moving all our DB queries to KV (what is KV we can have a separate discussion). In KV, we can't have join queries. So, at the end we have to remove all join queries so it's better to not add new ones.

Reply:
> Sure. Actually, we are moving all our DB queries to KV (what is KV we can have a separate discussion). In KV, we can't have join queries. So, at the end we have to remove all join queries so it's better to not add new ones.

Yeah, i know about this thing, that's why i'm not really complaining anymore about joins removing thing :D